### PR TITLE
Openvpn

### PR DESCRIPTION
**Pull request by mattiashem**
Openvpn update

### CODE REVIEW

File: bin/public/installOpenVPN.py [hunk 1]
```diff
@@ -71,9 +86,15 @@ def install_openvpn_server(args):
...
 
+    if len(args) != 2:
+        raise Exception("syco install-openvpn-server 2.3.7")
+
     # Initialize all passwords
     enable_ldap = config.general.get_option("openvpn.ldap.enable", "false")
-    x("yum -y install openvpn")
+    build_openvpn(args)
+    x('mkdir /etc/openvpn')
+    x('')
```
Comment (kribor) [inline bin/public/installOpenVPN.py]: I think these two lines that don't do anything can be removed...

File: bin/public/installOpenVPN.py [hunk 2]
```diff
@@ -135,8 +165,8 @@ def install_openvpn_server(args):
     if enable_ldap:
         _setup_ldap()
 
-    iptables.add_openvpn_chain()
-    iptables.save()
+    #iptables.add_openvpn_chain()
```
Comment (kribor) [inline bin/public/installOpenVPN.py]: If iptables is not set up here, where is it set up?

File: bin/public/installOpenVPN.py [hunk 3]
```diff
@@ -243,15 +273,16 @@ def _setup_ldap():
...
     general.set_config_property("/etc/cronjob", "01 * * * * root run-parts syco build_client_certs", "01 * * * * root run-parts syco build_client_certs")
 
     # Create client.conf
     clientConf = "/etc/openvpn/easy-rsa/keys/client.conf"
-    x("cp " + app.SYCO_PATH + "/var/openvpn/client.conf %s" % clientConf)
+    x("cp " + app.SYCO_PATH + "var/openvpn/client.conf %s" % clientConf)
     scOpen(clientConf).replace('${OPENVPN.HOSTNAME}',  config.general.get_openvpn_hostname())
 
-    x("cp " + app.SYCO_PATH + "/doc/openvpn/install.txt .")
+    #x("cp " + app.SYCO_PATH + "doc/openvpn/install.txt .")
```
Comment (kribor) [inline bin/public/installOpenVPN.py]: Copy from the actual path or remove this line

**Comment by kribor** [comment]
Since the way openvpn server installation is called is changing, all commandxx references to this must be updated. Both in mod-template of main syco and in private plugin repo's

File: bin/public/installOpenVPN.py (hunk 3 above)
Comment (mattiashem) [inline bin/public/installOpenVPN.py]: Finns inte mer så tog bort den

**Comment by mattiashem** [comment]
Updatertad instalaltionen.


- tog bort gamla yum av openvpn
- fixa iptables
- togbort gamal cp and install.txt fil

**Pull request merged**